composer.json

```diff
@@ -14,23 +14,24 @@
     "require": {
         "php": ">=7.2",
         "doctrine/doctrine-bundle": "^1.8|^2.0",
-        "doctrine/orm": "^2.6",
+        "doctrine/orm": "^2.7|^3.0",
         "league/oauth2-server": "^8.0",
         "psr/http-factory": "^1.0",
-        "sensio/framework-extra-bundle": "^5.3",
-        "symfony/framework-bundle": "^3.4|^4.2",
-        "symfony/psr-http-message-bridge": "^1.2",
-        "symfony/security-bundle": "^3.4|^4.2"
+        "sensio/framework-extra-bundle": "^5.5",
+        "symfony/psr-http-message-bridge": "^1.3",
+        "symfony/security-bundle": "^4.4|^5.0",
+        "symfony/framework-bundle": "^4.4|^5.0.2"
```

Comment:
What is the reason that 5.0.2 is picked as the minimum version in the 5.x series instead of just 5.0?

Reply:
fixed, my bad :) thank you!